server/src/app/ghost/ghost.service.ts

```diff
@@ -1,4 +1,28 @@
-import { Injectable } from '@nestjs/common';
+import {Injectable} from '@nestjs/common';
+import {IGhostRepository} from '../../domain/ghost/ghost.repository';
+import {Ghost} from '../../domain/ghost/ghost.entity';
 
 @Injectable()
-export class GhostService {}
+export class GhostService {
+  constructor(private readonly ghostRepository: IGhostRepository) {
+  }
+
+  async findOne(id: number): Promise<Ghost> {
+    return await this.ghostRepository.findOne(id);
+  }
+
+  async findAllByUser(userId: string): Promise<Ghost[]> {
+    return await this.ghostRepository.find(
+      {
+        userId,
+      },
+    );
+  }
+
+  async findAll(page: number): Promise<Ghost[]> {
+    return await this.ghostRepository.find({
+      take: 25,
+      skip: 25 * (page - 1),
```

Comment:
You implemented `skip: 25 * (page - 1)`, but in your test code, you tested with `findAll(0)`. In tc, `skip = -25`. Please check this.